src/classes/ADDR_AdminAcc_TEST.cls

```diff
@@ -0,0 +1,891 @@
+        }
+        ADDR_Addresses_TDTM.hasRunAddrTrigger = false;
+        Test.startTest();
+        insert addrs;
+        ADDR_Addresses_TDTM.hasRunAddrTrigger = false;        
+        Test.stopTest();
+
+        // verify Account address is new
+        map<Id, Account> mapAccIdAcc = new map<Id, Account>([select Id, Name, BillingStreet, BillingCity, BillingState, 
+        BillingPostalCode, BillingCountry, BillingLatitude, BillingLongitude, is_Address_Override__c, Current_Address__c from Account]);
+        
+        for (Account acc : mapAccIdAcc.values()) {
+            system.assertEquals('New Default Street\r\nSecond Line', acc.BillingStreet);
+            system.assertEquals('New Default Street\r\nSecond Line', acc.BillingStreet);
+        }
+        
+    }
+    
+    /*********************************************************************************************************
+    @description Update accounts' billing address with multiline street address  
+    verify:
+        new Default Address created
+        ACcounts get new Address
+        new Addresss has multiline street split into Street and Street2 fields
+    **********************************************************************************************************/            
+    @isTest
+    public static void updateMultilineStreetAddr() {
+        configSettings();
+        
+        // this creates a default Address for each Account
+        UTIL_UnitTestData_TEST.AccsWithAddrs accsAddrs = UTIL_UnitTestData_TEST.createTestAccsAddrs(2, UTIL_Describe.getAdminAccRecTypeID());
+        List<Address__c> addrs = accsAddrs.addrs;
+        
+        // modify some of the account addresses directly
+        for (Integer i = 0; i < accsAddrs.accs.size(); i++) {
+            Account acc = accsAddrs.accs[i];
+            acc.BillingStreet = 'Direct Street Edit\r\nSecond Line';
+            acc.BillingCity = 'Direct City Edit';
+        }
+        ADDR_Addresses_TDTM.hasRunAddrTrigger = false;
+        Test.startTest();
+        update accsAddrs.accs;
+        Test.stopTest();
+        
+        // verify Account address
+        map<Id, Account> mapAccIdAcc = new map<Id, Account>([select Id, Name, BillingStreet, BillingCity, BillingState, 
+        BillingPostalCode, BillingCountry, BillingLatitude, BillingLongitude, is_Address_Override__c, Current_Address__c from Account]);
+        
+        for (Account acc : mapAccIdAcc.values()) {
+            system.assertEquals('Direct Street Edit\r\nSecond Line', acc.BillingStreet);
+            system.assertEquals('Direct Street Edit\r\nSecond Line', acc.BillingStreet);
+            system.assertEquals(false, acc.is_Address_Override__c);
+            system.assertNotEquals(null, acc.Current_Address__c);
+        }
+ 
+        // verify the Address objects split street.
+        list<Address__c> listAddr = [select Id, MailingStreet__c, MailingStreet2__c, Formula_MailingStreetAddress__c, Default_Address__c from Address__c];
```

Comment:
If the table in this query contains more than 50'000 records a runtime exception will be thrown. Consider either introducing a `WHERE` clause or contain the number of records redurned, or adding a `LIMIT` clause to cap the maximum number of records returned.